RELEASE.md

```diff
@@ -0,0 +1,63 @@
+# Release Process
+
+Releases are published to NPM when `develop` is merged into `main` AND when the merged code contains a version bump in the `package.json`.
+
+## Create and publish a new release:
+
+### Bump version
+
+1. Checkout to a new version bump branch from `develop`.
+
+```bash
+git checkout develop
+git checkout -b bump-vX.X.X
+```
+
+2. Bump version with [standard-version](https://github.com/conventional-changelog/standard-version). This will automatically increase the version number and release notes and commit the changes in a release commit. To create specific versions such as a prerelease version, see below.
+
+```bash
+npm run version
```

Comment:
We should set up this script so it works in this lib. 
You can check https://www.npmjs.com/package/standard-version to see how we do it in the other libraries

Reply:
Also we can use yarn here